packages/react/src/exampleTypes/StepSlider.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+
+import '@testing-library/jest-dom'
+import userEvent from '@testing-library/user-event'
+import { act, render, screen } from '@testing-library/react'
+
+import StepSlider from './StepSlider'
+
+import testTree from './stories/testTree'
+import { renderWithProviders } from '../utils/test'
+
+describe('StepSlider', () => {
```

Comment:
💎 here we _could_ remove the quotes and make it a touch closer to the implementation by not using a string representation.
```suggestion
describe(StepSlider, () => {
```

Reply:
That's right! On my first shot I did this but received an unexpected error:
```
It must be a named class, named function, number, or string.
```

That given my constraints, didn't wrestling enough to solve xD

But I’m on It.

Reply:
Oooh I see why now, `contexturify` is the culprit .